src/Repository/AdherentRepository.php

```diff
@@ -178,6 +178,7 @@ public function countActiveAdherents(): int
             ->createQueryBuilder('a')
             ->select('COUNT(a.uuid)')
             ->where('a.status = :status')
+            ->andWhere('a.adherent = 1')
```

Comment:
[This test](https://github.com/EnMarche/en-marche.fr/blob/testsuite/tests/Repository/AdherentRepositoryTest.php#L54) was passing because fixtures about simples users (`LoadUserData`) weren't loaded.

This new condition fixes this.